gapic-generator-cloud/templates/cloud/helpers/presenters/method_presenter.rb

```diff
@@ -23,13 +23,13 @@ def initialize method
     @method  = method
   end
 
-  def api
-    service.api
-  end
-
-  def package
-    service.package
-  end
+  # def api
```

Comment:
Why is this chunk of code commented out?

Reply:
It should have been deleted. It exposes schema objects and we don't want that. I actually thought I did delete it...

Reply:
Yep, left a commit unpushed. Removed.